client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -358,3 +363,20 @@ export function getArrayOfFilteredTasks( tasks: Task[], flow: string | null ) {
 		}, [] as Task[] )
 	);
 }
+
+// If email is verified, filter out verify_email task
+// If email is not verified, disable all other tasks
+export function updateTasksBasedOnEmailVerification(
```

Comment:
`NB`:

Can we write a unit test for this functionality? It should be relatively easy and it can be in a follow up

Reply:
Yup, I was assuming I'd do this as a follow up.